plasmapy/formulary/tests/test_mathematics.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the mathematics.py module."""
+
+
+import numpy as np
+import pytest
+
+from plasmapy.formulary import mathematics as mathematics
+
+# TODO: Move the Fermi integral tests over to this file?
+
+
+def test_rot_a_to_b():
+
+    # Test one rotation
+    a = np.array([1, 0, 0])
+    b = np.array([0, 1, 0])
+    R = mathematics.rot_a_to_b(a, b)
+    correct = np.array([[0, 1, 0], [-1, 0, 0], [0, 0, 1]])
+    np.testing.assert_allclose(R, correct, atol=1e-6)
+
+    # Test the special case where the two vectors are parallel
+    a = np.array([1, 0, 0])
+    b = np.array([1, 0, 0])
+    R = mathematics.rot_a_to_b(a, b)
+    correct = np.identity(3)
+    assert np.allclose(R, correct, atol=1e-6)
+
+    # Test the special case where the two vectors are anti-parallel
+    a = np.array([1, 0, 0])
+    b = np.array([-1, 0, 0])
+    R = mathematics.rot_a_to_b(a, b)
+    correct = -np.identity(3)
+    assert np.allclose(R, correct, atol=1e-6)
```

Comment:
In practice you want to write your tests so if one fails, then the other will still run.  In how you have the test written here, if the first test fails then the last two never run.  The best way to accomplish this is to use [`pytest.mark.parametrize`](https://docs.pytest.org/en/stable/parametrize.html), which would look a little like...

```python
@pytest.mark.parametrize(
    "a, b, correct",
    [
        # case 1
        (a, b, correct),
        # case 2
        (a, b, correct),
        ...
    ],
)
def test_rot_a_to_b(a, b, correct):
    R = rot_a_to_b(a, b)
    assert np.allclose(R, correct)
```